.gitignore

```diff
@@ -1,5 +1,8 @@
 # Development
 
+## External grading
+/pl_ag_jobs
+
```

Comment:
This directory is typically not expected to be saved inside the repository, as per the docs. Keeping them in the repo can cause some weird behaviour of scripts that make assumptions about the content in the repo. I strongly encourage you to use a different location for this directory.

Reply:
Fair point; I should probably revert this and the `pyproject.toml` change. It may be worth updating the docs too though, I don't think the repo caveat was mentioned.

![image](https://github.com/PrairieLearn/PrairieLearn/assets/35978975/4585b25c-2870-4fc7-a359-d904f1ba7ad3)

From [here](https://prairielearn.readthedocs.io/en/latest/installing/#support-for-external-graders-and-workspaces). I took the "can live anywhere" to really mean anywhere (and I found within the repo to be most convenient).